pg.py

```diff
@@ -59,7 +62,8 @@ def updateCorrectors(self, coefficient, clearFineQuantities=True, MatrixValued=F
         if self.printLevel >= 2:
             print 'Setting up workers'
         if MatrixValued:
-            eccontrollerMatrixValued.setupWorker(world, coefficient, IPatchGenerator, k, clearFineQuantities, self.printLevel)
+            pass
+            #eccontrollerMatrixValued.setupWorker(world, coefficient, IPatchGenerator, k, clearFineQuantities, self.printLevel)
```

Comment:
This will need to be fixed when eccontrollerMatrixValued is available

Reply:
It's available now